test/fixtures/user/user.js

```diff
@@ -21,6 +21,7 @@ module.exports = () => {
       isMember: true,
       phone: '1234567890',
       email: 'abc@gmail.com',
+      createdAt: 1621025657,
```

Comment:
The values in fixtures are being used by our integration and unit tests to check for the expected behavior.
Can we also have this timestamp coming from the getUnixEpochTime function? We'll be then actually going close to the expected behaviour and our tests would be checking real-life case than having to run tests on hardcoded things.. Then the tests themselves won't make any sense..The tests would be checking on hardcoded values, they'll pass, developers become happy but when it goes to production, something might break because we didn't test on real life scenario, we checked on hardcoded values..

I hope you're getting what I'm trying to say 😅

Reply:
I've to get the time from the created utility method. Correct me if I am wrong.

Reply:
Yes